stock_picking_partner_note/models/stock_picking_note.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2024 Camptocamp (<https://www.camptocamp.com>).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+from odoo import fields, models
+
+
+class StockPickingNote(models.Model):
+    _name = "stock.picking.note"
+    _order = "name"
```

Comment:
Make sequence related to note_type_id.sequence

```suggestion
    _order = "sequence,name"
```